capstone/src/main/java/com/step902020/capstone/security/IdentityController.java

```diff
@@ -1,17 +1,49 @@
 package com.step902020.capstone.security;
 
+import com.step902020.capstone.Individual;
+import com.step902020.capstone.IndividualRepository;
+import com.step902020.capstone.Organization;
+import com.step902020.capstone.OrganizationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.context.SecurityContext;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.oauth2.jwt.Jwt;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 @RestController
 public class IdentityController {
 
+  @Autowired
+  private IndividualRepository individualRepository;
+
+  @Autowired
+  private OrganizationRepository organizationRepository;
+
   @GetMapping("identity")
   public String getIdentity(CurrentUser user) {
     return user.getEmail();
   }
+
+  @GetMapping("user-info")
+  public Object getUserType(CurrentUser user)  {
+    Organization organization = this.organizationRepository.findByEmail(user.getEmail()).orElse(null);
+    if (organization != null) {
+      return organization;
+    } else {
+      Individual individual = this.individualRepository.findByEmail(user.getEmail()).orElse(null);
+      if (individual != null) {
+        return individual;
+      } else {
+        Map<String, String> map = new HashMap<String, String>();
+        map.put("userType", "null");
```

Comment:
Because "null" has such a special meaning in Java and Javascript, I'd recommend using a custom type such as "unknown".